client/stylesheets/_toCat.scss

```diff
@@ -3302,6 +3310,18 @@ On click the images will be displayed at normal size to complete the effect
     float: right;
 } 
 
+.listingItemLocationString {
+    margin-top: 18px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered background-color, border-radius, margin-top, padding